student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,7 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+
+    const char * username = my_username();
+    TEST_ASSERT_EQUAL_STRING_MESSAGE("jackcenter", username,"Error: The username does not match!");
```

Comment:
@jackcenter missing validation of the config file here, my_username() does not read the conf file.